src/mutable_arithmetics.jl

```diff
@@ -80,9 +80,32 @@ end
 function JuMP.add_to_expression!(aff::PAE, new_coef, new_var::JuMP.VariableRef)
     JuMP.add_to_expression!(aff.v, new_coef, new_var)
 end
+function JuMP.add_to_expression!(aff::PAE, new_var::JuMP.VariableRef)
+    JuMP.add_to_expression!(aff.v, new_var)
+end
 function JuMP.add_to_expression!(aff::PAE, new_param::Parameter, new_coef)
     JuMP.add_to_expression!(aff.p, new_coef, new_param)
 end
+function JuMP.add_to_expression!(aff::PAE, new_param::Parameter)
+    JuMP.add_to_expression!(aff.p,new_param)
+end
 function JuMP.add_to_expression!(aff::PAE, new_coef, new_param::Parameter)
     JuMP.add_to_expression!(aff.p, new_coef, new_param)
-end
\ No newline at end of file
+end
+
+#= 
+
+Convert

```

Comment:
@joaquimg these convert methods are still missing in #45 and needed in PowerSimulations.

Reply:
Okay!
I will add the first for sure.
are you sure the second is needed?
a GAEp should never be generated, an manipulated externaly, so it might be bug.
Unless some of the ParameterJuMP functions need it.

Reply:
is there any example? or its to hard to create one?